rg_instructor_analytics/static/rg_instructor_analytics/js/instructor_analytics.js

```diff
@@ -28,22 +191,23 @@
         var $content;
         $content = $('.' + CSS_INSTRUCTOR_CONTENT);
         tabList = [
-            new Tab(
+            EnrollmentTab(
                 $content.find('#enrollment-stats-btn'),
                 $content.find('#section-enrollment-stats')),
-            new Tab(
+            ProblemTab(
                 $content.find('#problems-btn'),
                 $content.find('#section-problem')),
-            new Tab(
+            GradebookTab(
                 $content.find('#gradebook-btn'),
                 $content.find('#section-gradebook'))
         ];
         tabList.forEach(function (tab) {
             tab.button.click(function () {
-                toogleToTab(tab)
+                toggleToTab(tab)
             })
         });
-        toogleToTab(tabList[0])
+
+        toggleToTab(tabList[0]);
     });
 
 }).call(this);
```

Comment:
It seems that `})();` is fair enough.

Reply:
As for me, better, when callers will be pass as context for this function.